### PR TITLE
deleted LoginFunctionality

### PR DESCRIPTION
**Pull request by ercancivi**
deleted LoginFunctionalities.java class

### CODE REVIEW

**Comment by MedSdiri** [comment]
Azad

**Review by asuman75 — APPROVED** [review]
all done

**Review by avsank — APPROVED** [review]
Good.

**Review by ramizkazak — APPROVED** [review]

**Review by IrinaMakaryan — APPROVED** [review]
Looks good

**Review by asuman75 — APPROVED** [review]
well

File: src/test/java/com/trycloud/tests/base/LoginFunctionalities.java [hunk 1]
```diff
@@ -1,59 +0,0 @@
...
-        }
-
-    }
-
-    @AfterMethod
-    public void tearDownMethod(){
-        super.tearDownMethod();
-    }
-}
-
```
Comment (ramizkazak) [inline src/test/java/com/trycloud/tests/base/LoginFunctionalities.java:59]: done

**Review by ramizkazak — APPROVED** [review]

**Pull request closed without merging**